File: my-app/firebase.py
import firebase_admin
from firebase_admin import credentials, firestore

# 서비스 계정 키로 인증
cred = credentials.Certificate("firebase_key.json")

# firebase 앱 초기화 (인증 + 설정)
firebase_admin.initialize_app(cred)

# firestore 클라이언트 가져오기 (사용 가능)
db = firestore.client()

File: my-app/src/firebase.js
// src/firebase.js
import { initializeApp } from 'firebase/app';
import { getFirestore } from 'firebase/firestore';

const firebaseConfig = {
  apiKey: "AIzaSyCp46WkZR2nbSGtskNabK17beYDCciU-Es",
  authDomain: "food-chatbot-web.firebaseapp.com",
  projectId: "food-chatbot-web",
  storageBucket: "food-chatbot-web.appspot.com",
  messagingSenderId: "988267365222",
  appId: "1:988267365222:web:6ebc9d02cf4b86970afea7",
  measurementId: "G-HV94CE7JHE"
};

console.log("✅ Firebase 초기화 성공"); // ✅ 이 줄을 여기!

const app = initializeApp(firebaseConfig);
const db = getFirestore(app);

export { db };
